api.js

```diff
@@ -0,0 +1,19 @@
+const axios = require('axios');
+
+let getData = (url, token, param, callback) => {
```

Comment:
@davidltruong 
One suggestion: I know I set up the server to hold the token at the beginning, but when looking at the code, I think it's more consice and clean if we have our API token in this file instead of passing it as an argument to the getData function.
What do you think?

Reply:
@zxl3269117  makes sense to me. Was just following your lead on where the token was. Ill change it now

Reply:
@davidltruong 
It looks like errors will be logged but not fully resolved via the callback functions. Have you you tested any failures?
@mpmanzo @zxl3269117